### ISSUE TITLE
Add a configuration for what tools to display

### ISSUE THREAD

**Issue by mejackreed** [ProjectMirador/mirador-annotations#19]
Example: do not show multi-polygon button

**Comment by robcast** [comment]
Is there a real use case to configure the buttons to show, or should we just hide the inoperable functions?

Thinking from the point of customizing and embedding Mirador a configuration may be useful in any case.

**Comment by mejackreed** [comment]
Yes I think this would be a nice thing for adopters to be able to configure. For instance maybe they only want to direct users to use rectangle annotations.
